Microsoft Defender/microsoft-defender-for-endpoints/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,172 @@
+        "type": "detected attack"
+      }
+    ]
+  },
+  {
+    "value": "Verification of the certificate issued by {microsoft.defender.certificate.issuer}",
+    "conditions": [
+      {
+        "field": "event.dataset",
+        "value": "device_file_certificate_info"
+      },
+      {
+        "field": "microsoft.defender.certificate.issuer"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "file.x509.serial_number",
+        "target": "microsoft.defender.certificate.issuer",
+        "type": "issued by"
+      },
+      {
+        "source": "file.x509.serial_number",
+        "target": "microsoft.defender.certificate.signer",
+        "type": "signed by"
+      }
+    ]
+  },
+  {
+    "value": "{process.executable} executed on {host.name}",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "process.executable"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "host.name",
+        "target": "process.executable",
+        "type": "was executed on"
+      }
+    ]
+  },
+  {
+    "value": "{user.domain}\\{user.name} logged on {host.name}",
+    "conditions": [
+      {
+        "field": "event.dataset",
+        "value": "device_logon_events"
+      },
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "user.name"
+      },
+      {
+        "field": "user.domain"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "user.name",
+        "target": "host.name",
+        "type": "logged on"
+      }
+    ]
+  },
+  {
+    "value": "{user.name} logged on {host.name}",
+    "conditions": [
+      {
+        "field": "event.dataset",
+        "value": "device_logon_events"
+      },
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "user.name"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "user.name",
+        "target": "host.name",
+        "type": "logged on"
+      }
+    ]
+  },
+  {
+    "value": "{source.address} connected to {destination.address}",
+    "conditions": [
+      {
+        "field": "event.dataset",
+        "value": "device_network_events"
+      },
+      {
+        "field": "source.address"
+      },
+      {
+        "field": "destination.address"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "source.address",
+        "target": "destination.address",
+        "type": "was connected to"
+      }
+    ]
+  },
+  {
+    "value": "A task update is sheduled on {host.name}",
+    "conditions": [
+      {
+        "field": "event.dataset",
+        "value": "device_events"
+      },
+      {
+        "field": "action.type",
+        "value": "ScheduledTaskUpdated"
+      },
+      {
+        "field": "host.name"
+      }
+    ]
+  },
+  {
+    "value": "{microsoft.defender.observer.interface.type} interface {microsoft.defender.observer.interface.name} on {host.name} is {microsoft.defender.observer.interface.status}",
```

Comment:
I think that if you use this kind of fields (non ECS) in a smart description, you should modify your entries in `fields.yml` 

Example
https://github.com/SEKOIA-IO/intake-formats/blob/8cb0df409da2c1a313f86c54c8e401e8e2051022/Windows/windows/_meta/fields.yml#L1-L8

Reply:
I checked, and it seems that it's not necessary.